resources/public/js/publishing.js

```diff
@@ -7,6 +7,15 @@ var timerDelay = iosDetected ? 800 : 400;
 var $note, $action, $preview, $plain_password, $tableau;
 var backendTimer;
 
+document.addEventListener('DOMContentLoaded', function () {
+  marked.setOptions({
+    langPrefix: 'hljs lang-',
+    highlight: function (code) {
```

Comment:
I'm sorry, I confused myself — this is the client side, so no arrows please :) But `view.js`is on the server, so we are safe there.